### ISSUE TITLE
rust-analyzer does not support workspace/executeCommand

### ISSUE THREAD

**Issue by mohamed-abdelnour** [ms-jpq/coq_nvim#400]
Hello, @ms-jpq! Thank you for all your work; I recently switched to coq.nvim, and it's fantastic!

https://github.com/ms-jpq/coq_nvim/blob/f8ded8657ce20431eeca912c8e3b20bbdb5073c5/coq/lsp/requests/lsp.lua#L174

I'm getting the following error using coq.nvim with rust-analyzer:

```text
method workspace/executeCommand is not supported by any of the servers registered for the current buffer
```

Here are a couple of relevant issues on the rust-analyzer and Neovim repositories, respectively:

- rust-analyzer/rust-analyzer#1232
- neovim/neovim#15183

## Reproduction

Assuming cargo, rust, rust-analyzer, the built-in LSP client and coq.nvim are installed and configured:

- Create a project, and open `main.rs`.

```console
$ cargo new demo && cd demo
$ nvim src/main.rs
```

- Define a function, for example, `fn f(x: i32) {}`.
- Call `f` from `main` using completion, upon hitting enter on `f(…) [Function] 「LSP」` the error pops up.

![2021-11-24-11-14-16](https://user-images.githubusercontent.com/68501778/143211313-b94fb1c1-fac7-44e1-9c67-9189e94efb13.png)
![2021-11-24-11-14-20](https://user-images.githubusercontent.com/68501778/143211337-4fb0b507-edda-43b2-95be-818727b9c18a.png)

**Comment by iiYese** [comment]
Can confirm I am also having this.

**Comment by mohamed-abdelnour** [comment]
This has been fixed in 5e906a36c858f732623c2be0809c1e418f19821d; thank you, @ms-jpq!

@iiYese, can you confirm that you're not getting the error as of 5e906a36c858f732623c2be0809c1e418f19821d (or later)?

**Comment by mohamed-abdelnour** [comment]
I'm going to close this now. If anyone is still having trouble with this, feel free to add a comment here and I'll re-open the issue.
